data_structures/trie/radix_tree.py

```diff
@@ -1,3 +1,5 @@
+import unittest
```

Comment:
We (and the majority of the Python community) use pytest instead of unittest so all this boilerplate is not needed.

Can you please create a ___separate___ simple pull request that adds a test that proves that `word = ""` fails?  Once that is proven, we can determine how to proceed with this pull request.

Reply:
It fails on subseunt recursive calls not on direct addition of "". Please check the issue it is trying to fix. I have added the same word insertions in Unit Test as well